contracts/staking/GoodFundManager.sol

```diff
@@ -269,9 +229,9 @@ contract GoodFundManager is DAOContract {
 			//transfer ubi to avatar on sidechain via bridge
 			require(
 				token.transferAndCall(
-					bridgeContract,
+					nameService.addresses(nameService.BRIDGE_CONTRACT()),
 					gdUBI,
-					abi.encodePacked(ubiRecipient)
+					abi.encodePacked(nameService.addresses(nameService.UBISCHEME()))
```

Comment:
lets change UBISCHEME UBI_RECIPIENT
but keep also the UBISCHEME constant. it is used on Fuse contracts